packages/jasmine/src/jasmine_node_test.bzl

```diff
@@ -51,7 +55,13 @@ def jasmine_node_test(
         tags = tags,
     )
 
+    # append on any user args to the end
+    args = ["--coverage=%s" % coverage] + args
+
     all_data = data + srcs + deps + [jasmine]
+
+    # code coverage reporting dependancy
+    all_data += ["@npm//v8-coverage"]
```

Comment:
You can remove this line. Pulling in `@npm//@bazel/jasmine` (the `jasmine` attribute) is sufficient to also pull in `v8-coverage` since its a transitive dep. Also, we don't want to assume `@npm` is the users node_modules workspace. That's why we have the `jasmine` attribute so a user can set it to `@foobar//@bazel/jasmine` if necessary.

Reply:
I'm running into issues with this one.
If I remove it then im not able to run the tests inside the jasmine package anymore, since nothing is pulling `@npm//v8-coverage` anymore

Reply:
Yeah I would assume it is just an issue of getting the right package locally now so it pulls in the right deps automatically. I will take a look myself if I spot anything obvious.

Reply:
I think you just need to add `@npm//v8-coverage` explicitly to the deps (or data) of the test within `packages/jasmine/test/BUILD.bazel`:
```
jasmine_node_test(
    name = "coverage_test",
    srcs = [
        "coverage.spec.js",
        "coverage_source.js",
    ],
    coverage = True,
    jasmine = "@npm//jasmine",
    deps = [
        "//:jasmine_runner",
        "@npm//v8-coverage,
    ],
)
```

Reply:
ah! easy as, sorted